packages/nvidia_nat_core/src/nat/front_ends/fastapi/message_handler.py

```
@@ -405,8 +451,8 @@ async def create_websocket_message(self,
 
             message_schema: type[BaseModel] = await self._message_validator.get_message_schema_by_type(message_type)
 
-            if hasattr(data_model, 'id'):
-                message_id: str = str(getattr(data_model, 'id'))
+            if hasattr(data_model, "id"):
+                message_id: str = str(getattr(data_model, "id"))
```

Ruff (0.16.2)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)